accessCeramics/accessCeramics
# Feature/pyrometric cones (#2)

## testing notes
- try going to the admin backend (`localhost:8000/admin` by default) and check to make sure that cones have already been added (should happen when you run `python manage.py migrate`)
- try adding some new cones
- try making a new work and adding some cones to it
## dev notes
- adds a `PyrometricCone` model to the database that can store its cone number as a string
- adds the `PyrometricCone` model to the admin backend
- the `Work` model now has a `manyToMany` relationship with `PyrometricCone`
- adds a migration script to pre-populate the database with well-known cone numbers [taken from this pdf](https://www.ortonceramic.com/files/2676/File/orton-cone-chart-2016.pdf)
